.travis.yml

```diff
@@ -57,8 +57,10 @@ before_install:
     fi
   - if [[ "$DOC" == "true" ]]; then
       sudo apt-get install -qq asymptote texlive-latex-extra texlive-latex3 texlive-fonts-recommended;
-      sudo apt-get install -qq  texlive-font-utils latexmk texlive-xetex lmodern;
+      sudo apt-get install -qq texlive-font-utils latexmk texlive-xetex lmodern;
+      sudo apt-get install -qq graphviz;
       sudo add-apt-repository -y ppa:inkscape.dev/stable-daily && sudo apt-get update && sudo apt-get install -qq inkscape;
+      pip install pydot;
```

Comment:
I think this line is unnecessary.

Reply:
I have not added it, but OK, lets remove it...

Reply:
Yes, it was me, my bad 😅️